src/bgen/BindingTouch.cs

```diff
@@ -69,9 +69,11 @@ public class BindingTouch : IDisposable {
 
 	public MetadataLoadContext? universe;
 	public TypeManager? typeManager;
+	NamespaceManager? namespaceManager;
 	public TypeManager TypeManager => typeManager!;
 	public Frameworks? Frameworks;
 	public AttributeManager? AttributeManager;
+	public NamespaceManager NamespaceManager => namespaceManager!;
```

Comment:
nit but having the similar types grouped together makes it easier to follow

```suggestion
	public TypeManager? typeManager;
	public TypeManager TypeManager => typeManager!;
	public Frameworks? Frameworks;
	public AttributeManager? AttributeManager;
	public NamespaceManager NamespaceManager => namespaceManager!;
	NamespaceManager? namespaceManager;
```

Reply:
Made some of those changes in the next PR: https://github.com/xamarin/xamarin-macios/pull/19380